app/src/renderer/vuex/modules/index.js

```diff
@@ -1,14 +1,13 @@
-const files = require.context('.', false, /\.js$/)
-const funcs = {}
-files.keys().forEach((key) => {
-  if (key === './index.js') return
-  funcs[key.replace(/(\.\/|\.js)/g, '')] = files(key).default
+export default (opts) => ({
```

Comment:
require.context is not available in jest (comment on that from the lead dev of jest: https://github.com/facebookincubator/create-react-app/issues/517#issuecomment-243488178)